manager/Makefile

```diff
@@ -50,6 +50,11 @@ install: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl manifests
 uninstall: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl manifests
 	$(TOOLBIN)/kustomize build config/crd | $(TOOLBIN)/kubectl delete -f -
 
+# Install CRDs into a charts/m4d-crds
+.PHONY: charts-crd
+charts-crd: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl manifests
```

Comment:
Do we want to just put it in `manifests` directly? Otherwise the chart might accidently be outdated if someone pushed a PR that changes the CRDs but forgot to run charts-crd

Reply:
for now I don't want to enforce any automation, will do once I have a fully helm based deployment and then will change the integration test to be helm based (and then will add automation to ensure consistency, plus ensure that `make verify` ensures crds are up-to-date)

Reply:
OK, I'll need to remember this one.